interface/main/calendar/find_patient_popup.php

```diff
@@ -124,7 +124,7 @@
     echo "pflag=0"; } ?>'>
     <div class="form-row">
     <label for="searchby" class="col-form-label col-form-label-sm col"><?php echo htmlspecialchars(xl('Search by:'), ENT_NOQUOTES); ?></label>
-   <select name='searchby' id='searchby' class="form-control form-control-sm col">
+   <select name='searchby' id='searchby' class="form-control-sm col">
```

Comment:
This actually should be `form-control form-control-sm col`

Reply:
Hi @tywrenn but this class is causing the issue I mentioned.
If this is a mistake I will check and correct soon.

Reply:
I agree maybe fault is somewhere else. I will commit soon.